requirements.txt

```diff
@@ -1,2 +1,3 @@
-pbr>=0.11
+hatchling
+hatch_vcs
```

Comment:
You can put all these in `pyproject.yaml`.

Reply:
toml, not yaml ;-)

Can this be rebased now that the tests have been updated?

Reply:
It's been rebased.